src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/tts_get_capabilities_response_test.cc

```diff
@@ -133,6 +133,24 @@ TEST_F(TTSGetCapabilitiesResponseTest, Run_Nothing_SUCCESS) {
   command->Run();
 }
 
+TEST_F(TTSGetCapabilitiesResponseTest,
+       onTimeOut_Run_ResponseForInterface_ReceivedError) {
+  MessageSharedPtr msg = CreateMessage();
+  (*msg)[strings::params][hmi_response::code] =
+      hmi_apis::Common_Result::ABORTED;
+
+  std::shared_ptr<TTSGetCapabilitiesResponse> command(
+      CreateCommand<TTSGetCapabilitiesResponse>(msg));
+
+  EXPECT_CALL(
+      mock_hmi_capabilities_,
+      RemoveFromDefaultInitializedCapabilities(hmi_apis::FunctionID::TTS_GetCapabilities))
+      .Times(2);
+
+  command->Run();
```

Comment:
@ydementieiev This is the 3rd time when you receive comment about adding `Init()` for EVERY command before `Run()`. Check ALL your code. Nobody will write the same comment in hundred of places. You have to check and add required changes yourself.

Reply:
@AByzhynar add [everywhere](https://github.com/smartdevicelink/sdl_core/pull/3269/commits/a63d278ff6aea958f03c176be3f160a7dc6b2f24)